src/js/letters/utils/helpers.js

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable camelcase */
-
+import React from 'react';
 import merge from 'lodash/fp/merge';
 
 import environment from '../../common/helpers/environment';
+import { formatDateShort } from '../../post-911-gib-status/utils/helpers';
```

Comment:
Should we move this to a common place? Have been meaning to check if there are already helpers like this anywhere in `src/js/common`